.github/workflows/ghpages.yml

```
@@ -19,10 +19,10 @@ jobs:
     steps:
 
       - name: Checkout repository
-        uses: actions/checkout@v3
+        uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744
```

```suggestion
        uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3
```

We could also leave this with just the version tag since it's not a third party action

---

Oh shoot thanks for reminding me